backend/services/emailTemplates/conversation/text.ejs

```diff
@@ -0,0 +1,5 @@
+Video from <%= authorName %>:
+<%= title %>
+<%= message %>
+Follow the link to see the video: <%= buttonLink %>
+BizKonect - Video Messaging Platform for Technology Sales. Record, send & track one-to-one personalized video messages direct to your Customer’s Email, Social Media, or Text.
```

Comment:
Is it ok to start new line with '+' or it's better to place it on the line before?